routes/v1.js

```diff
@@ -81,7 +81,7 @@ module.exports = function(appObj) {
 
     return {
         path: '/',
-        api_version: 1,
```

Comment:
Yeah, the trick is that currently you can either keep both the domain and the version or skip them both. Should probably change that though.